python/paddle/v2/reader/creator.py

```diff
@@ -57,22 +57,51 @@ def reader():
     return reader
 
 
-def recordio(path):
+def recordio_local(paths):
     """
-    Creates a data reader that outputs record one one by one from given recordio file
-    :path: path of recordio file
-    :returns: data reader of recordio file
+    Creates a data reader that outputs record one one by one 
+        from given local recordio fils path.
+    :path: path of recordio files.
+    :returns: data reader of recordio files.
     """
 
     import recordio as rec
 
     def reader():
-        f = rec.reader(path)
+        for i, path in enumerate(paths):
+            f = rec.reader(path)
+            while True:
+                r = f.read()
+                if r is None:
+                    break
+                yield r
+            f.close()
+
+    return reader
+
+def recordio(paths, addr="", buf_size=100):
```

Comment:
I think `addr` need to be an environment variable, since user would never know the address of the etcd cluster. (so no way to provide this argument).